packages/components/src/popover/style.scss

```diff
@@ -1,7 +1,9 @@
 $arrow-triangle-base-size: 14px;
 
 .components-popover {
+	position: absolute;
```

Comment:
Why is that necessary? I thought `floating-ui` does its own job of deciding whether the element will be `absolute` or `fixed` (also see [strategy](https://floating-ui.com/docs/computeposition#strategy)).

Reply:
Not sure it's necessary, it's so that the browser has a base value before the JS adds it (avoid going from static to absolute|fixed)

Reply:
Same thoughts as @tyxla , the popover [should already render with `position: absolute`](https://github.com/WordPress/gutenberg/blob/6ab84d5ab63beaac3f630ed20ed53516b077ca7e/packages/components/src/popover/index.tsx#L490)

Reply:
👍 Fixed